src/python.c

```diff
@@ -1805,6 +1805,7 @@ datumToPython(Datum datum, Oid type, ConversionInfo * cinfo)
             return datumDecimalToPython(datum, cinfo);
         case DATEOID:
             return datumDateToPython(datum, cinfo);
+        case TIMESTAMPTZOID:
```

Comment:
I believe Postgres stores a timestamp with time zone after having shifted it by its timezone offset. Does that mean we're getting a timestamp without timezone, but shifted?